frontend/src/lib/Quiver.ts

```diff
@@ -715,9 +804,56 @@ but was expecting \`${JSON.stringify(expectedIndexTypes)}\`.
     return `${leftBracket + leftInterval}, ${rightInterval + rightBracket}`
   }
 
+  private static formatPeriodType(data: number, typeName: PeriodType): string {
+    const duration = Number(data)
```

Comment:
Why do we parse this as number if according to the function signature this already is a number?